### ISSUE TITLE
about AVA dataset

### ISSUE THREAD

**Issue by ArsenLuca** [titu1994/neural-image-assessment#10]
Hi @titu1994 , would you please share the AVA dataset files? I could not access the original AVA dataset page due to some reason in China mainland.

**Comment by titu1994** [comment]
It can be found on the websites for it, via the Reddit download links.

**Comment by ArsenLuca** [comment]
yes, I can find the website in Reddit, but due to some reason in China mainland, I can not open the links.

**Comment by titu1994** [comment]
Well, that's were I downloaded it from. Perhaps try the academic torrents website if it is available legally (I doubt it should be a problem) in China. I can't host this dataset since it is nearly 31 GB in size.

**Comment by ArsenLuca** [comment]
ok，I‘ll try other ways @titu1994

**Comment by tfriedel** [comment]
I've uploaded a version with images resized to 256x256 to [google drive](https://drive.google.com/file/d/1G_FcHMTv3a_GYq0vSzZK_t_1B9TSv2OG/view?usp=sharing). It's about 5.7 GB. 
Note the images are stored as jpg with 95% quality, so there are some artifacts present which you wouldn't have if you resized the original jpg images. I hope it will not matter much. 
I uploaded it to google drive because I'm trying out how feasible training on google's [colab](colab.research.google.com) platform is. They offer their version of Jupyter notebooks with free GPU.  An instance can run 12h.
